.changeset/kind-pens-argue.md

```diff
@@ -0,0 +1,6 @@
+---
+'@graphql-codegen/typescript-vue-urql': minor
```

Comment:
I think we should ship this as a breaking change cause I don't want users to have a surprise upgrading if we are making it more explicit
```suggestion
'@graphql-codegen/typescript-vue-urql': major
```

Reply:
The main breaking change in #328 was already released as I know, subscription part is not so widely used as I see from much less reactions, but lets do what is better for the project